etc/build.properties

```diff
@@ -1,10 +1,10 @@
 #
-# local.properties.example
-# ------------------------
-# system variables which will be used at build-time
-# Any of these values can be overwritten by copying
-# this file to etc/local.properties and manually
-# editing it.
+# build.properties
+# ----------------
+# System variables which will be used at build-time.
+# Any of these values can be overwritten by creating
+# a file called etc/local.properties and editing them
+# in this local file.
```

Comment:
Maybe slight rewording here?

Any of these values can be overwritten by creating a new file called etc/local.properties and then adding the variables to be overwritten to this new file.